cmd/kind/load/docker-image/docker-image.go

```diff
@@ -18,16 +18,12 @@ limitations under the License.
 package load
 
 import (
-	"os"
-	"path/filepath"
-
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
+	loadutils "sigs.k8s.io/kind/cmd/kind/load/load-utils"
```

Comment:
load logic should probably live in `pkg/`, importing kind as a library shouldn't require looking in `cmd` imo.

we can do something like pkg/cluster context.LoadImageTar and combine that with a util in pkg/container/docker to save tars

Reply:
Ideally we should probably refactor & document so that something like:
- pkg/cluster
- pkg/cluster/nodes
- pkg/cluster/config (and api versions)
- NOT pkg/cluster/\<anything else\>

Are used for library usage, and the rest are just used internally or are the commands themselves.

Reply:
Done. Ended up moving the functionality into /pkg/cluster/nodes to maintain the load cmd logic as is. Please tell me what you think